### ISSUE TITLE
Cannot download inswapper_128.onnx

### ISSUE THREAD

**Issue by osma** [s0md3v/roop#20]
I tried to install following the instructions in the README. There is a OneDrive link to a file that needs to be downloaded and as I understood it the file is called `inswapper_128.onnx`. But clicking the link gives me an error:

![image](https://github.com/s0md3v/roop/assets/1132830/954f71ac-513a-4116-8045-afd6561d85c5)

Maybe this file got too popular?

Trying to run the code gives this error:

    AssertionError: model_file inswapper_128.onnx should exist

Similar to #6 which was closed earlier.

**Comment by thakurudit** [comment]
I think the original model was deleted based on [this](https://github.com/deepinsight/insightface/issues/2294) thread.

Try this:-

`!gdown 1krOLgjW2tAPaqV-Bw4YALz0xT5zlb5HF`

source: [reddit](https://www.reddit.com/r/midjourney/comments/13pnraj/please_reupload_inswapper_128onnx/)

**Comment by osma** [comment]
Oh wow, I wasn't aware of the complicated story behind this model. Thanks @thakurudit , did the trick!

**Comment by s0md3v** [comment]
I have added mirrors in the README and will continue to add more. Thanks for heads up!

**Comment by LokeYourC3PH** [comment]
I will reupload this file as well as to add some alternative sources. I just managed to download it before it went down. That was damn fast too might I add, I downloaded it not too long ago from GDrive and now it's already down on OneDrive.

And lol, seems like the makers of said model want to lock it to their Discord, that's shady but whatever, we have it now and opposed to their statement, it works just fine.

@s0md3v Here a mirror: https://mega.nz/file/9l8mGDJA#FnPxHwpdhDovDo6OvbQjhHd2nDAk8_iVEgo3mpHLG6U
